src/content/docs/en/pages/guides/marketplace/templates/videoteca-static-cache.mdx

```diff
@@ -0,0 +1,96 @@
+description: Deploy an application with static cache settings using the Videoteca Player.
+meta_tags: templates, guides, Azion Marketplace, web
+namespace: docs_guides_static_cache_videoteca_player
+permalink: /documentation/products/guides/static-cache-videoteca-player/
+---
+
+import Tag from 'primevue/tag';
+
+<Tag severity="info" client:only="vue">
+Preview
+</Tag>
+
+The **Static Cache + Videoteca Player** template enhances the delivery and performance of static and dynamic content based on file extensions. It implements the Azion **Videoteca Player** integration for easy hosting of your Videofront's Videoteca player via iframe embedding. 
+
+The deployment of this template creates a GitHub repository containing your project, as well as an edge application with several Cache Settings and a preconfigured route to launch the Videoteca Player edge function, as well as a domain to facilitate your access and management through Azion Edge Platform.
+
+
+## Requirements
+
+- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
+    - Every push will be deployed automatically to this repository to keep your project updated.
+- The [Videoteca Player integration](/en/products/guides/videoteca-player/) installed and configured in your Azion account.
+- This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), [Image Processor](/en/documentation/products/build/edge-application/image-processor/) and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
+
+---
+
+## Deploying the template
+
+You can obtain and configure your template through the Azion Console. To easily deploy it at the edge, click the button below.
+
+<LinkButton
+    label="Deploy"
+    link="https://console.azion.com/create/azion-community/static-cache-videoteca-player"
+    icon="ai ai-azion"
+    icon-pos="left"
+  />
+
+---
+
+## Setting up the template
+
+In the configuration form, you must provide the information to configure your application. Fill in the presented fields. 
+
+Fields identified with an asterisk are mandatory.
+
+1. Define a name for your edge application.
+    - The bucket for storage and the edge function will use the same name.
+    - Use a unique and easy-to-remember name. If the name has already been used, the platform returns an error message.
+2. Fill the additional fields with the information needed for configuring the **Videoteca Player integration**.
+    - `Origin Address`: the domain name or IP address for your origin.
+    - `Custom Host Header`: customized value for the `host` header. Leave it blank to use the same address from the previous field.
```

Comment:
in other documentation we have this comment to complement: - Example: `${host}`. Read more about the host header and how to customize it on the [Origins](/en/documentation/products/edge-application/origins/#host-header) documentation. 

Maybe it can be useful here.